homeassistant/components/lidarr/sensor.py

```diff
@@ -0,0 +1,162 @@
+"""Support for Lidarr."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from copy import deepcopy
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Generic
+
+from aiopyarr import LidarrQueueItem, LidarrRootFolder
+
+from homeassistant.components.sensor import (
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import DATA_GIGABYTES
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+
+from . import LidarrEntity
+from .const import BYTE_SIZES, DOMAIN
+from .coordinator import LidarrDataUpdateCoordinator, T
+
+
+def get_space(data: list[LidarrRootFolder], name: str) -> str:
+    """Get space."""
+    space = []
+    for mount in data:
+        if name in mount.path:
+            mount.freeSpace = mount.freeSpace if mount.accessible else 0
+            space.append(mount.freeSpace / 1024 ** BYTE_SIZES.index(DATA_GIGABYTES))
+    return f"{space[0]:.2f}"
+
+
+def get_modified_description(
```

Comment:
Don't modify the description. That's not readable. We want the descriptions to be declaratively defined.

Add a suffix to the name and unique_id if needed in the entity instead.